workshops/image_editor/README.md

```diff
@@ -0,0 +1,451 @@
+1. Creates a variable that holds the text input that we placed in section 1.
+2. Creates another variable that will get the data gotten from the text input, we do this using the **value** property.
+3. Creates an if statement that checks if the variable `urlimageval` is empty or not using the **length** property, this property returns `true` if the length of the string is greater than 0 or `false` if the length of the string is 0. When the property returns true, it takes the `img` element in section 2 and sets its `src` to the value of `urlimage`, then it resets the value of `urlimage`.
+4. Then we use the `preventDefault();` method to prevent the default way the browser handles `submit` when it happens, which includes refreshing the page.
+5. Finally, we use the function `console.log('');` to send a message to the console.
+
+Now let's test it:
+
+![Image displaying gif](https://cloud-jumlfocs6.vercel.app/0screen_recording_2020-11-08_at_11.26.14_pm.gif)
+
+Works perfectly!
+
+Now we'll need to add an event listener to each range input in section 3, in order to be able to edit the image:
+```js
+for(let i=0; i<=inputrange.length-1; i++) {
+  inputrange[i].addEventListener('input', editimage);
+}
+```
+Let's break this down:
+In this code we are creating a for statement, in which we create a variable called `i` that starts at 0, on each iteration, we check if the condition is false or true, or in other words, whether `i` is smaller than the amount of elements in `inputrange`.
+
+- If the condition is true, for each input it adds an event listener to it, and when the event happens, the `editimage` function gets called.
+- If the condition is false, we stop the loop.
+
+ In order to get to the next iteration, we add one to `i` using `i++`.
+
+Now we'll create a function that takes the values from the range inputs, and then update the style of the image applying the css filter function using those values:
+
+```js
+function editimage() {
+  let gs = document.getElementById('gs');
+  let blur = document.getElementById('blur');
+  let huerotate = document.getElementById('hue-rotate');
+  let sepia = document.getElementById('sepia');
+
+  let gsval = gs.value;
+  let blurval = blur.value;
+  let huerotateval = huerotate.value;
+  let sepiaval = sepia.value;
+
+  targetimage.style.filter = 'grayscale('+gsval+'%) blur('+blurval+'px) hue-rotate('+huerotateval+'deg) sepia('+sepiaval+'%) invert('+invertval+'%)';
+}
+```
+Breaking down:
+1. The first 4 variables hold the range inputs from the HTML document using the method: ```document.getElementById('')``` 
+2. The another 4 variables take those range inputs and get them actual values, using the `value` property
+3. Then we take the `targetimage` element and we style it using the `style` & `filter` properties, and we specify what filters we want to apply.
+
+Let's test it:
+![Image displaying gif](https://cloud-rnyr55z84.vercel.app/0screen_recording_2020-11-08_at_11.31.24_pm.gif)
```

Comment:
Also should add an empty line between "Let's test it" and the image, because right now these two render next to each other instead of above/under